roles/openshift_inventory/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- no_log: True  # keep task descriptions legible
+  block:
+
+  - name: Standardize on latest variable names
+    set_fact:
+      deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
+      openshift_deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
+
+  - name: Normalize openshift_release
+    set_fact:
+      openshift_release: "{{ openshift_release | string | regex_replace('^v', '') }}"
+    when: openshift_release is defined
+
+- name: Ensure a valid deployment type has been given |
```

Comment:
I second that the ` |` is surprising. The "name" should be something semantically reasonable, a period would be better.


tl;dr

"Improving" the output by adding that suffix is not that great, it is like sticking \<font\> tags in HTML5.
For example, if we send a patch upstream to improve the appearance of tasks in the default callback plugin, we could end up getting something like `TASK ... been given || msg=...`. Also, we would be improving one message at the expense of lacking consistency with the rest.